### PR TITLE
[Refactor] Removing math_extensions.py

### PR DESCRIPTION
**Pull request by mfixstsci**
Addressing #68

### CODE REVIEW

**Comment by mfixstsci** [comment]
@bholler Here is the PR removing `math_extensionsx.py` from the package. We only imported a few constants and a couple of functions that I just added to the classes where they were called and used.

File: jwst_gtvt/rotationsx.py [hunk 1]
```diff
@@ -844,7 +896,7 @@ def cvt_c1c2_using_body2inertial_Q_to_v2v3pa_tuple(Q,coord1,coord2):
    Vp_eci.set_xyz_from_angs(coord1,coord2)
    Vp_body_pt = Q.inv_cnvrt(Vp_eci)
    v2 = atan2(Vp_body_pt.y,Vp_body_pt.x)
-   v3 = asin(unit_limit(Vp_body_pt.z))
+   v3 = anp.sin(unit_limit(Vp_body_pt.z))
```
Comment (bholler) [inline jwst_gtvt/rotationsx.py:899]: A few lines from 809 to 899 have "anp.sin" or "math.anp.sin," and I can't find those when doing a quick Google search. Should they actually be "np.asin"?

Comment (mfixstsci) [reply jwst_gtvt/rotationsx.py:899]: @bholler thank you! So there are so many calls to `sin` and `cos` from the python native `math` library and when I did a search and replace, I figured there might be some lines that will cause bugs 😓 this is why I need you to review it for me!

This is also one of the reasons we need to do this, there is so much of this code base that isn't even touched by the examples/use cases this tool is used for. Who knows, we may never even touch this line of code.

I can go ahead and fix it for now, but we might end up removing it down the road!

**Review by bholler — APPROVED** [review]
Did another quick check through the changes and I don't see any other bad replacements.

File: jwst_gtvt/rotationsx.py (hunk 1 above)
Comment (mfixstsci) [reply jwst_gtvt/rotationsx.py]: @bholler should be able to see changes in [a328c03](https://github.com/spacetelescope/jwst_gtvt/pull/75/commits/a328c030e43382d41bde4f8d49f6011164403299).

**Pull request merged**